modules/core/src/main/scala-2/sup/HealthResultTaglessInstances.scala

```diff
@@ -0,0 +1,11 @@
+package sup
+
+import cats.tagless.FunctorK
+import cats.~>
+
+trait HealthResultTaglessInstances {
```

Comment:
Strange, I thought cats-tagless-core would be available for Scala 3 (it doesn't have macros AFAIK). Maybe it's worth making a ticket to see if/when that could happen :)

Reply:
There's a somewhat stale issue suggesting that it requires some CI updates first (and macros are blocked by Scala itself): https://github.com/typelevel/cats-tagless/issues/170